### PR TITLE
add grpc proto of pump service

### PR DESCRIPTION
**Pull request by iamxy**
@coocood @shenli
PTAL

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: proto/binlog/pump.proto [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    uint64 clusterID = 1;
+
+    // Payload bytes can be decoded back to binlog struct by the protobuf.
+    bytes payload = 2;
+}
+
+message WriteBinlogResp {
+    // An empty errmsg returned means a successful write.
+    // Otherwise return the error description.
+    string errmsg = 1;
```
Comment (shenli) [inline proto/binlog/pump.proto]: Add  a error code?

Comment (iamxy) [inline proto/binlog/pump.proto]: It's a very simple RPC method, return a nil for success just like func in golang.
I think it's clear and simple enough in the early stage. 
However, when the caller needs to be treated according to the return value, an error code is necessary.

**Comment by shenli** [comment]
LGTM

**Pull request merged**